themes/default/content/docs/intro/languages/go.md

```diff
@@ -10,7 +10,7 @@ menu:
 
 <img src="/logos/tech/logo-golang.png" align="right" width="150" style="padding:8px; margin-top: -64px">
 
-Pulumi supports writing your infrastructure as code using the Go language. Go 1.16 or later is required.
+Pulumi supports writing your infrastructure as code in the Go language built for any [Supported versions](https://go.dev/doc/devel/release#policy).
```

Comment:
```suggestion
Pulumi supports writing your infrastructure as code in the Go language built for any [supported versions](https://go.dev/doc/devel/release#policy).
```

Reply:
I added a tweak to @squaremo's suggestion to make `support version` singular (i.e., "for any any supported version").